src/locales/ko/mystery-encounters/fiery-fallout-dialogue.json

```
@@ -1,26 +1,26 @@
 {
-  "intro": "You encounter a blistering storm of smoke and ash!",
-  "title": "Fiery Fallout",
-  "description": "The whirling ash and embers have cut visibility to nearly zero. It seems like there might be some... source that is causing these conditions. But what could be behind a phenomenon of this magnitude?",
-  "query": "What will you do?",
+  "intro": "매캐한 연기와 화산재의 폭풍을 맞닥뜨렸다!",
+  "title": "불타는 폴아웃",
+  "description": "소용돌이치는 화산재와 불씨로 인해 앞이 거의 보이질 않습니다.\n이런 데는 무언가… 원인이 있는 것 같습니다.\n그렇지만 이정도로 큰 규모라면 뒤에 도대체 뭐가 있는 걸까요?",
```

```suggestion
  "description": "소용돌이치는 화산재와 불씨로 인해 앞이 거의 보이질 않습니다.\n이런 일에는 무언가… 원인이 있는 것 같습니다.\n그렇지만 이 정도로 큰 규모라면 뒤에 도대체 뭐가 있는 걸까요?",
```
조금 더 명확하게 수정하여 제안합니다.